iothub_client/samples/iotedge_downstream_device_sample/iotedge_downstream_device_sample.c

```diff
@@ -0,0 +1,313 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please call out explicitly towards the top that this sample is for IoT Edge specifically and put a link to IoT Edge's GitHub repro (or wherever you point folks) for additional info RE Edge.